cache/sizedlru/sizedlru.go

```diff
@@ -0,0 +1,111 @@
+// Package sizedlru implements an LRU cache that bounds the total space occupied
+// by the items it contains.
+
+package sizedlru
```

Comment:
Maybe just call it lru?

Reply:
I actually moved it into the main package.